src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,70 @@
-import React from 'react';
+import { FC, ChangeEvent, useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard/MovieCard';
+
+type Props = {
+  movies: Movie[];
+  handleMovieSet: (value: Movie) => void;
+};
+
+export const FindMovie: FC<Props> = (
+  {
+    movies,
+    handleMovieSet,
+  },
+) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [title, setTitle] = useState('');
+  const [isFindMovie, setFindMovie] = useState(false);
+  const [isError, setError] = useState(false);
+
+  const onInputChange = (event: ChangeEvent<HTMLInputElement>) => {
+    event.preventDefault();
```

Comment:
```suggestion
    event.preventDefault();
```
What default behaviour do you want to prevent?